hack/build-image.sh

```diff
@@ -64,5 +65,5 @@ for IMAGE_TO_BUILD in $TOBUILD; do
 	NAME="${IMAGE_TO_BUILD#Dockerfile.}"
   NAME="${NAME//.upstream}"
 	set -x
-	$podman build -t "localhost/${NAME}:latest" -f "${IMAGE_TO_BUILD}" --no-cache
+	$podman build -t "localhost/${NAME}:latest" -f "${IMAGE_TO_BUILD}" 
```

Comment:
Uhm, what about the changes in this file

Reply:
Not directly related to the patch. But this lets a developer build more quickly the image. Maybe I'll send it as a separate productivity patch?